Examples/Tests/reduced_diags/analysis_reduced_diags.py

```diff
@@ -76,90 +91,86 @@
+values_yt['maximum of rho'] = np.amax(rho)
+values_yt['minimum of rho'] = np.amin(rho)
+values_yt['electrons: maximum of rho'] = np.amax(np.abs(rho_electrons))
+values_yt['protons: maximum of rho'] = np.amax(np.abs(rho_protons))
+
+#--------------------------------------------------------------------------------------------------
+# Part 2: get results from reduced diagnostics (label '_rd')
+#--------------------------------------------------------------------------------------------------
+
+# Quantities computed from reduced diagnostics
+values_rd = dict()
+
+# Load data from output files
+EFdata = np.genfromtxt('./diags/reducedfiles/EF.txt')
+EPdata = np.genfromtxt('./diags/reducedfiles/EP.txt')
+MFdata = np.genfromtxt('./diags/reducedfiles/MF.txt')
+MRdata = np.genfromtxt('./diags/reducedfiles/MR.txt')
+NPdata = np.genfromtxt('./diags/reducedfiles/NP.txt')
+
+# First index "1" points to the values written at the last time step
+values_rd['field energy'] = EFdata[1][2]
+values_rd['particle energy'] = EPdata[1][2]
+values_rd['maximum of Ex'] = MFdata[1][2]
+values_rd['maximum of Ey'] = MFdata[1][3]
+values_rd['maximum of Ez'] = MFdata[1][4]
+values_rd['maximum of |E|'] = MFdata[1][5]
+values_rd['maximum of Bx'] = MFdata[1][6]
+values_rd['maximum of By'] = MFdata[1][7]
+values_rd['maximum of Bz'] = MFdata[1][8]
+values_rd['maximum of |B|'] = MFdata[1][9]
+values_rd['maximum of rho'] = MRdata[1][2]
+values_rd['minimum of rho'] = MRdata[1][3]
+values_rd['electrons: maximum of rho'] = MRdata[1][4]
+values_rd['protons: maximum of rho'] = MRdata[1][5]
+values_rd['number of particles'] = NPdata[1][2]
+values_rd['electrons: number of particles'] = NPdata[1][3]
+values_rd['protons: number of particles'] = NPdata[1][4]
+values_rd['photons: number of particles'] = NPdata[1][5]
+values_rd['sum of weights'] = NPdata[1][6]
+values_rd['electrons: sum of weights'] = NPdata[1][7]
+values_rd['protons: sum of weights'] = NPdata[1][8]
+values_rd['photons: sum of weights'] = NPdata[1][9]
+
+#--------------------------------------------------------------------------------------------------
+# Part 3: compare values from plotfiles and reduced diagnostics and print output
+#--------------------------------------------------------------------------------------------------
+
+error = dict()
+tolerance = 1e-12
+
+print()
+for k in values_yt.keys():
+    print('values_yt[' + k + '] = ', values_yt[k])
+    print('values_rd[' + k + '] = ', values_rd[k])
+    error[k] = abs(values_yt[k] - values_rd[k]) / abs(values_yt[k])
+    print('relative error = ', error[k])
+    print()
+    # The comparison of field energies requires a large tolerance,
+    # possibly because the field energy from the plotfiles is computed from cell-centered data,
+    # while the field energy from the reduced diagnostics is computed from (Yee) staggered data.
+    # NOTE The current relative error in this case is around 25%: might need further investigation.
```

Comment:
I would suggest to add `field_energy_tolerance = 0.3` right after `tolerance = 1e-12` 
and move the comment before the `for k in values_yt.key()` cycle.
Then we could do
`    tol = field_energy_tolerance if (k == 'field energy') else tolerance`

btw, 25%? that's quite a lot for field energy... You are right: this requires further investigation